subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt

```diff
@@ -167,14 +168,16 @@ private fun parseSentryConfig(argumentsProvider: ArgsProvider): SentryConfig {
     val tags = mapOf(
         "API" to Build.VERSION.SDK_INT.toString()
     )
+    val release = BuildMetadata.androidLibVersion("test-inhouse-runner")
```

Comment:
A client has to point out exact module because it's possible to use different versions of different dependencies. 
We use the same infra version in publishing but don't forbid this use case.


```groovy
androidTestImplementation("com.avito.android:snackbar-proxy:2021.11")
androidTestImplementation("com.avito.android:proxy-toast:2021.10")
```

```kotlin
androidLibVersion("snackbar-proxy") -> "2021.11"
androidLibVersion("proxy-toast") -> "2021.10"
```
Otherwise, it will be a clash and incorrect version.